### PR TITLE
Nb of genes

### PR DESCRIPTION
**Pull request by lldelisle**
- add the number of genes above which label is switched to off as a parameter (max_labels) relative to #26.

### CODE REVIEW

File: pygenometracks/tests/test_bed_and_gtf_tracks.py [hunk 1]
```diff
@@ -98,3 +128,16 @@ def test_plot_tracks_bed_and_gtf():
...
     os.remove(outfile.name)
+
+
+def test_plot_tracks_bed_with_maxLab():
+
+    outfile = NamedTemporaryFile(suffix='.png', prefix='pyGenomeTracks_test_', delete=False)
+    args = "--tracks {0}/bed_maxLab_tracks.ini --region X:2000000-3500000 --trackLabelFraction 0.2 --width 38 " \
+           "--dpi 130 --outFileName  {1}".format(ROOT, outfile.name).split()
+    pygenometracks.plotTracks.main(args)
+    print("saving test to {}".format(outfile.name))
```
Comment (bgruening) [inline pygenometracks/tests/test_bed_and_gtf_tracks.py]: this can be removed I think

Comment (lldelisle) [reply pygenometracks/tests/test_bed_and_gtf_tracks.py]: You mean you would not do a test for this functionality so I remove the test, the ini file and the png?

Comment (LeilyR) [reply pygenometracks/tests/test_bed_and_gtf_tracks.py]: I wouldn't remove any test.  Let's see what Bjoern thinks, maybe he found it redundant with another test?

Comment (bgruening) [reply pygenometracks/tests/test_bed_and_gtf_tracks.py]: Oh no tests are great, I love tests, but the print is not needed right?

**Pull request merged**